@xen-orchestra/fs/src/_mount.js

```diff
@@ -37,38 +38,43 @@ export default class MountHandler extends LocalHandler {
   }
 
   async _forget() {
-    await this._execa('umount', ['--force', this._getRealPath()], {
-      env: {
-        LANG: 'C',
-      },
-    }).catch(error => {
-      if (
-        error == null ||
-        typeof error.stderr !== 'string' ||
-        !error.stderr.includes('not mounted')
-      ) {
-        throw error
-      }
-    })
+    const keeper = this._keeper
+    if (keeper === undefined) {
+      return
+    }
+    await fs.close(keeper)
+
+    await ignoreErrors.call(
```

Comment:
Why `ignoreErrors` here ?

Reply:
Because:
- handling errors is difficult: we have to parse an error message which may depends on the driver used and the `mount` implementation
- we want to ignore errors due to another handler/process using the mount (via a keeper file)
- there is nothing the user can do in case of an error except ignore or display it

Reply:
Ok, thanks :)